components/brave_wallet/browser/tx_service.cc

```diff
@@ -48,21 +57,37 @@ size_t CalculatePendingTxCount(
 
 }  // namespace
 
-TxService::TxService(JsonRpcService* json_rpc_service,
-                     BitcoinWalletService* bitcoin_wallet_service,
-                     KeyringService* keyring_service,
-                     PrefService* prefs)
+TxService::TxService(
+    JsonRpcService* json_rpc_service,
+    BitcoinWalletService* bitcoin_wallet_service,
+    KeyringService* keyring_service,
+    PrefService* prefs,
+    const base::FilePath& context_path,
+    const scoped_refptr<base::SequencedTaskRunner>& ui_task_runner)
```

Comment:
nit: instead of 
`const scoped_refptr<base::SequencedTaskRunner>& ui_task_runner`
it is suggested to use this form https://chromium.googlesource.com/chromium/src/+/lkgr/styleguide/c++/c++.md#object-ownership-and-calling-conventions
`scoped_refptr<base::SequencedTaskRunner> ui_task_runner`

Reply:
fixed in f253205c8cf8d7ec8b87d2e88d4c355d76e4eb7f